docs/results.md

```diff
@@ -0,0 +1,376 @@
+Currently the following measurements are extracted:
+* static compile (mean)
+* average borrow time
+* num free jrubies
+
+These measurements are extracted for each timestamp in the metrics data and stored in the `puppetserver.csv` file.
+The average value for each measurement across the timestamps is calculated and stored in the `puppetserver.average.csv` file.
+As with other CSV files in the results these are converted into HTML documents: `puppetserver.csv.html` and `puppetserver.average.csv.html`.
+
+## Tune settings
+The current values for the settings available for tuning via the [pe_tune](https://github.com/tkishel/pe_tune) module are captured in the `current_tune_settings.json` file.
+
+
+---
+
+
+# Scale
+Scale tests consist of multiple iterations of a Performance scenario. 
+Actually, a copy of the scenario JSON file is created for each iteration with the number of agents set based on the increment value set for the Scale run.
+Each copy of the scenario is then run as a separate Performance test.
+
+The results for Scale test runs can be found in the [`results/scale`](https://github.com/puppetlabs/gatling-puppet-load-test/tree/master/results/scale) directory.
+The results for each Scale test run are contained within a separate timestamped sub-directory of the `results/scale` directory.
+The name of each result directory consists of the label `SCALE_` to indicate a Scale run followed by a timestamp generated at the start of the run.
+The directory also contains a `latest` link pointing to the latest Scale test results.
+
+```
+results/scale
+├── PERF_SCALE_1572446292
+└── latest -> /home/centos/gplt/slv-680/gatling-puppet-load-test/results/scale/PERF_SCALE_1572446292
+
+```
+
+Each Scale test result directory contains directories and files related to the overall Scale test run as well as each iteration of the scenario.
+For example:
+```
+results/scale/PERF_SCALE_1572446292
+├── PERF_SCALE_1572446292.csv
+├── PERF_SCALE_1572446292.csv.html
+├── Scale_1572446292_1_600
+├── Scale_1572446292_2_700
+├── Scale_1572446292_3_800
+├── beaker_environment.txt
+├── json
+├── log
+├── pe_tune_current.txt
+└── puppet-metrics-collector
+```
+
+## Scale test report
+The Scale test results contains a CSV file with measurements extracted from the Gatling JSON data for each iteration.
+This report is useful for comparing the performance impact of additional nodes for each iteration.
+It is also useful for comparing the outcome of multiple Scale test runs.
+
+In the example above this file is `PERF_SCALE_1572446292.csv`; 
+as with other CSV files in the results it has been converted to an HTML document (`PERF_SCALE_1572446292.csv.html`).
+
+## Beaker environment settings
+The `beaker_environment.txt` file contains the environment variables for the test run.
```

Comment:
:bulb: Future enhancement: We should add this to the performance tree as well.

Reply:
Updated SLV-692 to include this suggestion.